docs/blockchain/contracts/nft-contract/nft-tables.experimental.md

```diff
@@ -0,0 +1,812 @@
+| resale_shares           | std::vector\<eosio::resale_share> | A vector of [account, share] pairs setting the share each account receives during the Uniq resale.                                      |
+| mintable_window_start   | std::optional\<uint32_t>          | The beginning of the time window when tokens can be minted.                                                                             |
+| mintable_window_end     | std::optional\<uint32_t>          | The end of the time window when tokens can be minted.                                                                                   |
+| trading_window_start    | std::optional\<uint32_t>          | The beginning of the time window when tokens can be traded.                                                                             |
+| trading_window_end      | std::optional\<uint32_t>          | The end of the time window when tokens can be traded.                                                                                   |
+| recall_window_start     | std::optional\<uint32_t>          | *Disabled*. The beginning of the time window when tokens can be recalled.                                                               |
+| recall_window_end       | std::optional\<uint32_t>          | *Disabled*. The beginning of the time window when tokens can be recalled.                                                               |
+| lockup_time             | std::optional\<uint32_t>          | *Disabled*. The time window since Uniq minting in which the Uniq cannot be transferred                                                  |
+| conditionless_receivers | std::vector\<eosio::name>         | A set of Uniq receiver account tokens can be transferred to without any restrictions - like trading windows, minimum resell price, etc. |
+| stat                    | uint8_t                           | The Uniq factory status:0 = active - fully functional1 = inactive - cannot mint2 = shutdown - cannot mint or set active                 |
+| factory_uri             | std::string                       | The Uniq factory metadata URI vector.                                                                                                   |
+| factory_hash            | eosio::checksum256                | The Uniq factory metadata hash.                                                                                                         |
+| max_mintable_tokens     | std::optional\<uint32_t>          | The maximal number of tokens that can be minted with the factory.                                                                       |
+| minted_tokens_no        | uint32_t                          | The number of minted of tokens.                                                                                                         |
```

Comment:
Description should be 
The number of minted Uniqs.

Reply:
updated